src/pkg/utils/general_test.go

```
@@ -32,9 +41,79 @@ func TestContainsMention(t *testing.T) {
+	time.Sleep(1 * time.Second) // Allow server to start
+
+	meta := utils.GetMetaDataFromURL("http://localhost:8080")
+	assert.Equal(suite.T(), "Test Title", meta.Title)
+	assert.Equal(suite.T(), "Test Description", meta.Description)
+	assert.Equal(suite.T(), "http://example.com/image.jpg", meta.Image)
+}
+
+func (suite *UtilsTestSuite) TestDownloadImageFromURL() {
+	// Mock HTTP server
+	http.HandleFunc("/image.jpg", func(w http.ResponseWriter, r *http.Request) {
+		w.Write([]byte("image data"))
```

golangci-lint (1.62.2)

Error return value of `w.Write` is not checked

(errcheck)